packages/eslint-config-nimble/lib/index.js

```diff
@@ -2,28 +2,28 @@
  * @fileoverview eslint config
  * @author Nimbl3
  */
-"use strict";
+'use strict';
 
 module.exports = {
-  "parserOptions": {
-    "ecmaVersion": 9,
-    "ecmaFeatures": {
-      "jsx": true,
-      "impliedStrict": true
+  'parserOptions': {
```

Comment:
Should we remove the quotes from all keys?

```suggestion
  parserOptions: {
```

Reply:
@carryall I add this rule in [this PR](https://github.com/nimblehq/eslint-config-nimble/pull/34/files#diff-89f1a03a6c19194dc6d65422b78001b428457835090a723b7d79bc590eaf9a53R411), and fix them in that PR as well